### PR TITLE
OSSM-6378 xrefs in ossm-routing-ingress

### PR DESCRIPTION
**Pull request by gwynnemonahan**
[OSSM-6378](https://issues.redhat.com//browse/OSSM-6378) xrefs in ossm-routing-ingress

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSSM-6378

Link to docs preview:
https://76744--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-traffic-manage#ossm-routing-ingress_traffic-management 

QE review:

QE is not required as this is a style fix.

Additional information:
Request from Observability team related to their PR https://github.com/openshift/openshift-docs/pull/74819 and it is also a style fix.

### CODE REVIEW

File: service_mesh/v2x/ossm-traffic-manage.adoc [hunk 1]
```diff
@@ -19,6 +19,11 @@ endif::openshift-rosa,openshift-dedicated[]
 
 include::modules/ossm-routing-ingress.adoc[leveloffset=+2]
 
+ifdef::openshift-enterprise[]
+.Additional resources
+You can access the gateway using the service's xref:../../networking/configuring-node-port-service-range.adoc#configuring-node-port-service-range[node port].
```
Comment (mburke5678) [inline service_mesh/v2x/ossm-traffic-manage.adoc:24]: As an additional reference, shouldn't this be:

```suggestion
You can access the gateway using the service's xref:../../networking/configuring-node-port-service-range.adoc#configuring-node-port-service-range[node port].
```

File: service_mesh/v2x/ossm-traffic-manage.adoc [hunk 2]
```diff
@@ -19,6 +19,12 @@ endif::openshift-rosa,openshift-dedicated[]
 
 include::modules/ossm-routing-ingress.adoc[leveloffset=+2]
 
+ifdef::openshift-enterprise[]
+.Additional resources
+
+* xref:../../networking/configuring-node-port-service-range.adoc#configuring-node-port-service-range[node port].
```
Comment (adellape) [inline service_mesh/v2x/ossm-traffic-manage.adoc:25]: Link text should be the title of the target, also no puncutation:
```suggestion
* xref:../../networking/configuring-node-port-service-range.adoc#configuring-node-port-service-range[Configuring the node port service range]
```

**Review by adellape — COMMENTED** [review]
Holding merge-review for a nit, will keep merge-review labels on and watch for update. Thanks!

**Review by adellape — APPROVED** [review]

File: service_mesh/v2x/ossm-traffic-manage.adoc [hunk 3]
```diff
@@ -19,6 +19,12 @@ endif::openshift-rosa,openshift-dedicated[]
 
 include::modules/ossm-routing-ingress.adoc[leveloffset=+2]
 
+ifdef::openshift-enterprise[]
+.Additional resources
```
Comment (adellape) [inline service_mesh/v2x/ossm-traffic-manage.adoc:23]: Oops I'm sorry, I forgot about this bit:
```suggestion
[role="_additional-resources"]
.Additional resources
```

**Review by adellape — APPROVED** [review]

**Pull request merged**